nodejs/week1/avg.js

```diff
@@ -0,0 +1,12 @@
+// Excluding "node" and script name
+const args = process.argv.slice(2);
+
+// Convert to numbers
+const numbers = args.map(Number).filter(n => !isNaN(n));
+
+if (numbers.length>0) {
+    console.log(numbers.reduce((a, b) => a + b, 0) / numbers.length);
```

Comment:
Nice use of the `reduce` function! I do think that saving the result into a variable before printing would make the code more readable though.